source/mapping-rules/mapping-rule-options/embedded-documents.txt

```diff
@@ -1,3 +1,257 @@
+.. _rm-embedded-documents:
+
 ==================
 Embedded Documents
-==================
\ No newline at end of file
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The :guilabel:`Embedded Documents` mapping rule is an advanced mapping 
+rule option which must be explicitly added to a mapping model.
+
+About this Task
+---------------
+
+The :guilabel:`Embedded Documents` mapping rule is available if the relational
+table it is mapping from is on the one side of the foreign key relationship.
+
+The table which contains the foreign key must also be mapped to a 
+collection in the MongoDB model.
+
+Behavior
+--------
+
+Use the :guilabel:`Embedded Documents` mapping rule to denormalize
+a foreign key relationship. With :guilabel:`Embedded Documents` you can
+nest child foreign key fields in a parent collection.
+
+You have two main options when denormalizing your relational data 
+with the :guilabel:`Embedded Documents` option:
+
+- You can embed the child documents into the parent and designate a 
+  :guilabel:`Prefix` and :guilabel:`Field name` to embed fields as sub documents.
+
+- You can merge into the parent, which represents the rows as 
+  fields at the parent level.
+
+Examples
+--------
+
+This section demonstrates the relational input and
+the MongoDB output of the :guilabel:`Embedded Documents` mapping rule.
+
+Embedded Documents
+~~~~~~~~~~~~~~~~~~
+
+In this example:
+
```

Comment:
(praise)
This is a really nice use of a list to describe the example behavior. It helps, IMO, to have that setup in advance of the toggle.